install.sh

```diff
@@ -0,0 +1,2 @@
+chmod +x install
```

Comment:
Doesn't need this file

Reply:
Really? But some users won't know how to use chmod. Please think of it again.